financial-connections/src/main/java/com/stripe/android/financialconnections/di/FinancialConnectionsSheetSharedModule.kt

```diff
@@ -60,6 +62,10 @@ internal interface FinancialConnectionsSheetSharedModule {
     @Singleton
     fun bindsAnalyticsRequestV2Executor(impl: DefaultAnalyticsRequestV2Executor): AnalyticsRequestV2Executor
 
+    @Binds
+    @Singleton
```

Comment:
nit - doesn't need to be on the shared module (it can live on the native one) and can be a non-singleton as well